### DISCUSSION TITLE
posts/Algorithm-JS/

### DISCUSSION THREAD

**Discussion by giscus** [da-in/da-in.github.io#8] [General]
# posts/Algorithm-JS/

Programmers JavaScript 풀이 시작에 앞서.

https://da-in.github.io/posts/Algorithm-JS/

**Comment by Junwen-Zheng** [comment]
Great blog.
I think you can  put it in a file called sample.js and run it by writing: node sample.js
if you want to put it in the node environment
And then use process.stdin.
